gapic/generator/generator.py

```diff
@@ -167,7 +169,7 @@ def _generate_samples_and_manifest(
 
         # Only generate a manifest if we generated samples.
         if output_files:
-            manifest_fname, manifest_doc = samplegen.generate_manifest(
+            manifest_fname, manifest_doc = manifest.generate_manifest(
```

Comment:
How is the manifest time passed in?

Reply:
It isn't; it's an optional parameter with a default value of `None`, which means "use system time". The parameter is given a non-default value just for unit tests.

Reply:
Oh, I see, the `None` value causes the system time. OK.

One stylistic note: In case this app ever decides to output more than one timestamped file at a time, it would be good to have a time value obtained once (globally or at module init), so that all timestamped outputs can use it and label artifacts as  coming from the same invocation.

Reply:
Okay. I did it this way because it made unit testing easier, but mocking `time` does the same thing. I'll keep that in mind if more than one manifest needs to be generated per run.